CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,120 @@
+# Code of Conduct
```

Comment:
Rather than write our own, can we follow what sig-security is and just follow our governing body's COC?  Here is the link to the CNCF one:  https://github.com/cncf/foundation/blob/main/code-of-conduct.md

Reply:
First sentence: We follow the [CNCF Code of Conduct][cncf-coc].
How much more we should use the CNCF CoC?

The first part explains how we want to follow this commitment. The second part clarifies incident handling which the CoC doesn't because every project has to manage this by itself.

The next part is about references, which are not managed in CoC.